sdk/storage/AzureStorageBlob/Source/Transport/URLSessionTransferManager+ResumableOperationQueueDelegate.swift

```diff
@@ -30,6 +30,7 @@ extension URLSessionTransferManager: ResumableOperationQueueDelegate {
     internal func operation(_ operation: ResumableOperation?, didChangeState state: TransferState) {
         saveContext()
         guard let transfer = operation?.transfer else { return }
+        let delegate = client(forRestorationId: transfer.clientRestorationId)
```

Comment:
Ultimately, the delegate needs to be a UIViewController that implements TransferDelegate (or something that has a reference to such a controller) so that the UI can be updated. Does the AzureSDKSwiftDemo app work with this change?

Reply:
It does - this is because `StorageBlobClient` itself implements `TransferDelegate`, and it has a `delegate` property which is the actual delegate the customer supplies. The TransferManager calls the delegate method on the client, and the client in turn calls the delegate method on whatever the value of its `delegate` property is, if present. I missed committing the fixes to the demo app, but it's just a matter of updating the call sites that use the restorationID in the public API.